scripts/components/reportsDashboards/build.sh

```diff
@@ -61,6 +61,31 @@ fi
 
 [ -z "$OUTPUT" ] && OUTPUT=artifacts
 
+[ -z "$PLATFORM" ] && PLATFORM=$(uname -s | awk '{print tolower($0)}')
+[ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
+
+case $PLATFORM in
+    linux*)
+        PLATFORM="linux"
+        ;;
+    *)
+        echo "Unsupported platform: $PLATFORM"
+        exit 1
+        ;;
+esac
+
+case $ARCHITECTURE in
+    x64|arm64)
+        CHROMIUM_TARGET="chromium-$PLATFORM-$ARCHITECTURE.zip"
+        ;;
+    *)
+        echo "Unsupported architecture: $ARCHITECTURE"
+        exit 1
+        ;;
+esac
+
+CHROMIUM_URL="https://github.com/opensearch-project/dashboards-reports/releases/download/chromium-1.12.0.0/$CHROMIUM_TARGET"
```

Comment:
1.12.0 should be passed from $VERSION.

Reply:
this `chromium-1.12.0.0` is a fixed tag we used to put compiled chromium binaries. we won't release new chromiums in every opensearch release

Reply:
I think this is static endpoint. https://github.com/opensearch-project/dashboards-reports/releases/tag/chromium-1.12.0.0
Not something that is generated per version

Reply:
Version would be like OpenSearch Dashboards plugin version.